### ISSUE TITLE
Problem of override default `__eq__` in Panel

### ISSUE THREAD

**Issue by louxiu** [njean42/kumiko#21]
``` python
	def __eq__(self, other):
		return all(
			[
				abs(self.x - other.x) < self.wt(),
				abs(self.y - other.y) < self.ht(),
				abs(self.r - other.r) < self.wt(),
				abs(self.b - other.b) < self.ht(),
			]
		)
```

Override `__eq__` of `Panel` cause problem. Assume panel_a is `__eq__` to `panel_b` and `panel_b` is `__eq__` to `panel_c`, but it is may not true that `panel_a` is `__eq__` to `panel_c`.
If `panel_a`, `panel_b`, `panel_c` are in panels array(order a b c) and we remove it with.order `panel_b`, `panel_c`,  `panel_a`, then `panel_a`, `panel_b` will be remove in order, and error will occur when try to remove `pane_a` because `panel_c` is left in panels array. This may happens in `merge_panes` in page.py

Maybe we should avoid override default `__eq__` (__lt__) etc and use explicit func with `similar` `top_left` etc

**Comment by njean42** [comment]
Hey!
Have you encountered a situation, an image that causes an issue with the problem you're describing?
I'm not sure I understand what you mean here.

**Comment by louxiu** [comment]
> Hey! Have you encountered a situation, an image that causes an issue with the problem you're describing? I'm not sure I understand what you mean here.

Yes, this problem will cause exception like item not in list. I will check whether I can provide a sample image or a solution over the weekend.
